js/levels.js

```diff
@@ -25,7 +25,7 @@ var levels = [
       'uk': '<p>Вітаємо у Flexbox Froggy - гра, де ви маєте допомогати жабеняті Фрогі та його друзям написанням CSS коду! Всади жабеня на лист латаття праворуч, за допомогою властивості <code>justify-content</code>, яка вирівнює елементи по горизонталі й набуває таких значень:</p><ul><li><code>flex-start</code>: Елементи вирівнюються по лівій стороні контейнера.</li><li><code>flex-end</code>: Елементи вирівнюються по правій стороні контейнера.</li><li><code>center</code>: Елементи вирівнюються по центру контейнера.</li><li><code>space-between</code>: Елементи буде зображено з рівними відступами поміж них.</li><li><code>space-around</code>: Елементи буде зображено з рівними відступами навколо них.</li></ul><p>Наприклад, <code>justify-content: flex-end;</code> пересуне жабеня праворуч.</p>',
-      'hi': '<p>आपका Flexbox Froggy में स्वागत है। इस खेल में आप CSS Code लिख कर Froggy और उसके दोस्तों की मदद करते हैं। <code>justify-content</code> प्रॉपर्टी की सहायता से मेंढकों को लिली के पत्ते तक पहुंचाइये। यह प्रॉपर्टी वस्तुओं को छैतिज रूप से संरेखित (पंक्तिबद्ध ) करती है एवं निम्न मान स्वीकार करती है। :</p><ul><li><code>flex-start</code>: यह वस्तु को कंटेनर की बाईं तरफ सरेंखित करता है।</li><li><code>flex-end</code>: यह वस्तु को कंटेनर की दाईं तरफ सरेंखित करता है। </li><li><code>center</code>: यह वस्तु को कंटेनर के मध्य में संरेखित करता है। </li><li><code>space-between</code>: यह वस्तुओं को बराबर अंतर के साथ प्रदर्शित करता है। </li><li><code>space-around</code>: यह वस्तुओं के चारों ओर के अंतर को प्रदर्शित करता है।</li></ul><p>उदाहरण के लिये, <code>justify-content: flex-end;</code> मेंढकों को दायीं तरफ पंहुचा देगा।</p>',
+      'hi': '<p>आपका Flexbox Froggy में स्वागत है। इस खेल में आप CSS Code लिख कर Froggy और उसके दोस्तों की मदद करते हैं। <code>justify-content</code> प्रॉपर्टी की सहायता से मेंढकों को लिली के पत्तियों तक पहुंचाइये। यह प्रॉपर्टी वस्तुओं को छैतिज रूप से संरेखित (पंक्तिबद्ध ) करती है एवं निम्न मान स्वीकार करती है। :</p><ul><li><code>flex-start</code>: यह वस्तु को कंटेनर की बाईं तरफ सरेंखित करता है।</li><li><code>flex-end</code>: यह वस्तु को कंटेनर की दाईं तरफ सरेंखित करता है। </li><li><code>center</code>: यह वस्तु को कंटेनर के मध्य में संरेखित करता है। </li><li><code>space-between</code>: यह वस्तुओं को बराबर अंतर के साथ प्रदर्शित करता है। </li><li><code>space-around</code>: यह वस्तुओं के चारों ओर के अंतर को प्रदर्शित करता है।</li></ul><p>उदाहरण के लिये, <code>justify-content: flex-end;</code> मेंढकों को दायीं तरफ पंहुचा देगा।</p>',
```

Comment:
Can we change `के पत्तियों` to `की पत्तियों ` or `के पत्तों` like other change below. 
Other than this, it LGTM.

Reply:
`के पत्तों` will be good i think  👍🏽